flags/src/main/java/com/yahoo/vespa/flags/PermanentFlags.java

```diff
@@ -367,6 +367,12 @@ public class PermanentFlags {
             "Takes effect on next redeployment",
             INSTANCE_ID);
 
+    public static final UnboundBooleanFlag ALLOW_STATUS_PAGE = defineFeatureFlag(
+            "allow-status-page", false,
+            "Shows link to status page for nodes of a specific tenant",
```

Comment:
"status page" is a little vague, this should at least mention that this is the "cluster controller status page"

Reply:
I updated the description and renamed the flag now